app/services/maat/decision_translator.rb

```diff
@@ -56,28 +51,11 @@ def court_type
     def funding_decision
       return if means.blank?
 
-      case court_type
-      when Types::CourtType['crown']
-        crown_court_decision
-      when Types::CourtType['magistrates']
+      if maat_decision.cc_rep_decision.present?
+        CrownCourtDecisionTranslator.translate(maat_decision.cc_rep_decision)
+      else
         FundingDecisionTranslator.translate(maat_decision.funding_decision)
       end
     end
-
-    # Store the overall result in the format previously shown to eForms users.
-    # Storing the string since rules as to what is shown are complex and vary
-    # by court and case type.
-    #
-    # Note: The MAAT API returns a string for Crown Court decisions
-    # but a constant for Magistrates' Court decisions.
-    def overall_result
-      return maat_decision.cc_rep_decision if court_type == Types::CourtType['crown']
-
-      OverallResultTranslator.translate(maat_decision.funding_decision)
-    end
-
```

Comment:
☝️ we calculate the overall result dynamically using the OverallResultCalculator